css/secondary-page.styl

```diff
@@ -40,6 +40,16 @@
           font-weight: 400
           margin-bottom: 1em
 
+          img.avatar
+            padding-right: 10px !important
+
+          span.login-name
+            padding-left: 7px !important
+
+            span
+              font-size: 0.65em !important
+              font-weight: 200 !important
+
```

Comment:
I know that `!important` should be avoided, but I am not sure how to reorder things with Stylus such that these aren't needed, which is what I would do in normal CSS... I am assuming we don't want any repetition of classes, nor to change the order in which the `.styl` files are loaded... so am open to suggestions on how I can move these to a different point in the CSS order .... how to do this? add a new `.styl` file?